config/hw_bridge.h

```diff
@@ -4,27 +4,57 @@
 namespace ROVERCONFIG {
 constexpr uint16_t ROVER_MOTOR_PWM_FREQ_HZ = 20000;   // 20 kHz
 constexpr uint32_t ROVER_CANBUS_FREQUENCY  = 500000;  // 500 kbps
+constexpr uint16_t ROVER_CANID_FILTER_MASK = 0xFE0;   // Use bits [5:10] for addressing and 0:7 for command/message type
+}  // namespace ROVERCONFIG
+
+namespace CANFILTER {
 enum filtering_t {
-  ROVER_CANID_FILTER_MASK         = 0xFE0,  // Use bits [5:10] for addressing and 0:7 for command/message type
   ROVER_CANID_FIRST_ERROR_TX      = 0x100,
-  ROVER_CANID_FIRST_SAFETY_TX     = 0x720,
-  ROVER_CANID_FIRST_SAFETY_RX     = 0x730,
   ROVER_CANID_FIRST_ARM_RX        = 0x740,
   ROVER_CANID_FIRST_ARM_TX        = 0x750,
   ROVER_CANID_FIRST_SCIENCE_RX    = 0x760,
   ROVER_CANID_FIRST_SCIENCE_TX    = 0x770,
   ROVER_CANID_FIRST_GIMBTONOMY_RX = 0x780,
   ROVER_CANID_FIRST_GIMBTONOMY_TX = 0x790
 };
-}  // namespace ROVERCONFIG
+}
 
-namespace CANID {
-enum canID_t {
+namespace CANERROR {
+enum canError_t {
   // ERROR CAN IDs
   SAFETY_ERROR = 0x100,
   ARM_ERROR,
   SCIENCE_ERROR,
-  GIMBTONOMY_ERROR,
+  GIMBTONOMY_ERROR
+};
+}  // namespace CANERROR
+
+namespace CANHEARTBEAT {
+enum canHeartbeat_t {
+  // ERROR CAN IDs
+  JETSON = 0x140,
+  SAFETY,
+  ARM,
+  SCIENCE,
+  GIMBTONOMY,
+  ROBOTEQ = 0x701
+};
+}  // namespace CANHEARTBEAT
+
+namespace CANID {
+enum canID_t {
+  // DRIVETRAIN CAN IDs
+  TPDO1 = 0x181,
```

Comment:
@akeaveny could u modify the drivetrain code to use these structs once these prs are in